### PR TITLE
 fix incorrect timezone in log files

### PR DESCRIPTION
**Pull request by SadeghPM**
| Q             | A
| ------------- | ---
| Fixed ticket | #175 

Crunz timezone in log files is not depending on timezone that is used to calculate task run time.
Because crunz logs my task related event. I set timezone for my tasks, and crunz uses this timezone to run task, thus any logs that related to this task, should be log in configured timezone.

### CODE REVIEW

**Comment by PabloKowalczyk** [comment]
Hello @SadeghPM, i think that your code will not work because you do not add new entry to `\Crunz\Configuration\Definition` file. If you add `-vvv` to command you will see is config file was used.
Could copy `timezone_log` to `Configuration` section in README?

File: src/Logger/LoggerFactory.php [hunk 1]
```diff
@@ -10,9 +10,21 @@ class LoggerFactory
...
     private $configuration;
 
+    /**
+     * LoggerFactory constructor.
+     *
+     * @param Configuration $configuration
+     *
+     * @throws \Exception if the timezone supplied in configuration is not recognised as a valid timezone
+     */
     public function __construct(Configuration $configuration)
```
Comment (PabloKowalczyk) [inline src/Logger/LoggerFactory.php]: Could add unit test for new behavior?

Comment (SadeghPM) [reply src/Logger/LoggerFactory.php]: Test added https://github.com/lavary/crunz/pull/178/commits/5c06d5f443e04c9296d740bd99cd5c397e98c043

**Comment by SadeghPM** [comment]
Other thing like `readme.md` https://github.com/lavary/crunz/pull/178/commits/26e781fa03fd5e33f026c116a1567c79e903b02f  & `definition` https://github.com/lavary/crunz/pull/178/commits/60d76fcc25c490af8f7ae37b4402b3cff9b35f75 update done.

File: src/Configuration/Definition.php [hunk 2]
```diff
@@ -36,6 +36,10 @@ public function getConfigTreeBuilder()
                     ->info('Timezone used to calculate task run date')
                 ->end()
 
+                ->scalarNode('timezone_log')
```
Comment (PabloKowalczyk) [inline src/Configuration/Definition.php]: Why not `booleanNode` with default `false` value?

**Comment by PabloKowalczyk** [comment]
BTW i will merge this PR after stable `v1.11` release, because new features should go to `v1.12`, not `v1.11` as it is in RC stage.

File: src/Configuration/Definition.php (hunk 2 above)
Comment (SadeghPM) [reply src/Configuration/Definition.php]: done:https://github.com/lavary/crunz/pull/178/commits/4470a8aa42da207a05cce5e873d5480cf7f0be8f

**Comment by SadeghPM** [comment]
> BTW i will merge this PR after stable `v1.11` release, because new features should go to `v1.12`, not `v1.11` as it is in RC stage.
I think It looks more like a bug than a new feature. But as you wish.

**Comment by PabloKowalczyk** [comment]
Adding new configuration option looks like new feature :)

**Comment by PabloKowalczyk** [comment]
Thanks @SadeghPM

**Comment by SadeghPM** [comment]
Thanks crunz contributors for the best crunz package in php world!

**Pull request merged**